apps/full-site-editing/README.md

```diff
@@ -0,0 +1,54 @@
+# Full Site Editing
+
+This app contains both the `full-site-editing-plugin` and the required `blank-theme`.
+
+## File Architecture
+
+These is an example of what we should find inside the theme and plugin folders:
+
+```
+/blank-theme
+  /dist
+    blank-theme.css
+    blank-theme.js
+  functions.php
+  index.js
+  index.php
+  index.scss
+
+/full-site-editing-plugin
+  /dist
+    full-site-editing-plugin.css
+    full-site-editing-plugin.js
+  full-site-editing-plugin.php
+  index.js
+  index.scss
+```
+
+## Local Development
+
+Symlink both the theme and the plugin into a local WordPress install.
+
+E.g.
+
+```
+ln -s ~/Dev/wp-calypso/apps/full-site-editing/full-site-editing-plugin/ ~/Dev/wordpress/wp-content/plugins/full-site-editing-plugin
+
+ln -s ~/Dev/wp-calypso/apps/full-site-editing/blank-theme/ ~/Dev/wordpress/wp-content/themes/blank-theme
+```
+
+## Build System
+
+There are 4 scripts that perform roughly the same tasks, but on different folders and environments:
+
+```
+npm run dev:plugin
```

Comment:
Note that you can also run these scripts from the Calypso root dir using lerna and the `scope` param: `npx lerna run dev:plugin --scope='@automattic/@automattic/full-site-editing'`.

Actually, this will be the recommended way included on the monorepo docs @simison is currently working on #32137: https://github.com/Automattic/wp-calypso/blob/42496d0976a6ea54fd4204daaedbdd12615141c4/apps/README.md

Reply:
I'm not sure if there are any negative consiquences running `npm` commands (especially `npm install` in `/apps/**` subfolders instead of running them from the root using `scope`.

I didn't add running `npm` scripts from subfolders to docs just because I didn't consider someone would want to do that. :-) I don't really want to recommend the way or the other without understanding Lerna better.

Maybe it'll skip some `pre*` commands and dependencies don't get hoisted properly between this folder and root `node_modules`? Can that lead to inconsistent versions depending what you run? Since you use `^` range in semver, it's prolly not a concern. cc @blowery who knows this stuff better.